### DISCUSSION TITLE
melonDS bios question

### DISCUSSION THREAD

**Discussion by parknich082** [EmulatorJS/EmulatorJS#381] [Q&A] [answered]
okay so im trying to use the melonds core, and im not sure how i am supposed to enter the bios url, do i use an array? i have the firmware.bin bios7.bin and bios9.bin but idk how to enter them in properly, thanks in advance

**Comment by ethanaobrien** [comment, marked as answer]
Just include the path to the file, same as you do the game url

**Reply by parknich082** [reply]
yeah but theres 3 files..?

**Reply by ethanaobrien** [reply]
Zip them together, the js code will unzip the files into the base directory and it will be as if you had them all separately.

You can zip them in the rar, 7z, or zip format

**Reply by parknich082** [reply]
> Zip them together, the js code will unzip the files into the base directory and it will be as if you had them all separately.
> 
> You can zip them in the rar, 7z, or zip format

ohhhh okay ty

**Reply by ethanaobrien** [reply]
No problem

**Reply by parknich082** [reply]
> No problem

hm, i put the biospath as "https://notarealwebsite.???/emujs/rom/bios/nds_bios.zip" the nds_bios.zip containing "firmware.bin", "bios7.bin", and "bios9.bin". but now when i get "Network Error" in bold red text after it decompresses the game core. any idea?

**Reply by ethanaobrien** [reply]
Well the url needs to be a real website. If you put the file in the same directory as your html file it can just be the file name. You could also use the emulatorjs code helper to do it all for you

**Reply by parknich082** [reply]
the "notarealwebsite.???" was just a placeholder...

**Reply by parknich082** [reply]
and the "code helper" doesnt let me insert a bios

**Reply by parknich082** [reply]
okay i think i got it to work, but the game im trying to use is 128mb but i need it to be under 100mb, its a .nds, do you know how i can reduce the size? cause i tried just having it as a .zip but it seems to get stuck on "Download Game Data"

**Reply by ethanaobrien** [reply]
You can also have it compressed in the 7z or rar compression formats

**Reply by parknich082** [reply]
> You can also have it compressed in the 7z or rar compression formats

i cant do that on a chromebook

**Reply by ethanaobrien** [reply]
Then you should be working on your work during school, not playing games
